test/qpu_interface.jl

```diff
@@ -506,24 +506,36 @@ end
 end
 
 
-@testset "AbstractConnectivity: excluded_positions" begin
+@testset "AbstractConnectivity: excluded positions and connections" begin
 
     excluded_positions = [1, 2, 3, 9, 10]
+    excluded_connections = [(2, 3), (5, 4)]
```

Comment:
let's use `[(2,3),(6,5)]`, so that we can also test a path between 2 valid qubits that are both not subject to an excluded position